src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilterMutectCalls.java

```diff
@@ -25,9 +25,10 @@
  * Filter SNVs and indels from a Mutect2 callset.
  *
  * <p>
- *     FilterMutectCalls encapsulates GATK3 MuTect2's filtering functionality.
+ *     FilterMutectCalls encapsulates GATK3 MuTect2's filtering functionality and adds additional filters.
  *     GATK4 Mutect2 retains variant calling and some prefiltering.
- *     Separating calling and filtering functionalities into two tools better enables an iterative filtering process
+ *     Thresholds for filters are contained in {@link M2FiltersArgumentCollection} and described in docs/mutect/mutect.pdf.
```

Comment:
The M2FiltersArgumentCollection has no meaning to forum doc readers. We will need to reveal these thresholds somehow either via code translation to a new tool doc (e.g. like the annotation or read filters classes) and/or a forum article. I think we should consult @vdauwera and @cmnbroad on this matter.

Change doc path to absolute link:
--> https://github.com/broadinstitute/gatk/tree/master/docs/mutect/mutect.pdf

Reply:
> Change doc path to absolute link

done.